src/marketing/price-rules-cart-coupon.md

```diff
@@ -73,33 +73,111 @@ The generation of discount coupons is an asynchronous operation, which executes
 
 1. To limit the number of times each customer can use the coupon, enter the number of **Uses per Customer**.
 
-    ![Cart price rule - generate auto-numbered coupons]({% link images/images/price-rule-cart-coupon-auto.png %}){: .zoom}
-    _Generate Auto-numbered Coupons_
+   ![Cart price rule - generate auto-numbered coupons]({% link images/images/price-rule-cart-coupon-auto.png %}){: .zoom}
+   _Generate Auto-numbered Coupons_
 
 1. Scroll down and expand ![Expansion selector]({% link images/images/btn-expand.png %}) the **Manage Coupon Codes** section. Then, do the following:
 
-    ![Cart price rule - manage coupon codes]({% link images/images/price-rule-cart-manage-coupon-codes.png %}){: .zoom}
-    _Manage Coupon codes_
-
-   - In the **Coupons Qty** field, enter the number of coupons that you want to generate.
+   ![Cart price rule - manage coupon codes]({% link images/images/price-rule-cart-manage-coupon-codes.png %}){: .zoom}
+   _Manage Coupon codes_
 
-   - Enter the **Code Length**, not including the prefix, suffix, or separators.
+   -  In the **Coupons Qty** field, enter the number of coupons that you want to generate.
 
-   - Set the **Code Format** to one of the following:
+   -  Enter the **Code Length**, not including the prefix, suffix, or separators.
 
-      - `Alphanumeric`
-      - `Alphabetical`
-      - `Numeric`
+   -  Set the **Code Format** to one of the following:
 
-   - (Optional) Enter a **Code Prefix** to be added to the beginning of the code.
+      -  `Alphanumeric`
+      -  `Alphabetical`
+      -  `Numeric`
 
-   - (Optional) Enter a **Code Suffix** to be added to the end of the code.
-
-   - (Optional) In the **Dash Every X Characters** field, enter the number of characters between each dash. For example, if the code is twelve characters long, and there is a dash every four characters, it will look like ` xxxx-xxxx-xxxx`. Dashes make codes easier to read and enter.
+   -  (Optional) Enter a **Code Prefix** to be added to the beginning of the code.
+   -  (Optional) Enter a **Code Suffix** to be added to the end of the code.
+   -  (Optional) In the **Dash Every X Characters** field, enter the number of characters between each dash. For example, if the code is twelve characters long, and there is a dash every four characters, it will look like ` xxxx-xxxx-xxxx`. Dashes make codes easier to read and enter.
```

Comment:
```suggestion
   -  (Optional) For **Dash Every X Characters**, enter the number of characters between each dash. For example, if the code is twelve characters long, and there is a dash every four characters, it will look like ` xxxx-xxxx-xxxx`. Dashes make codes easier to read and enter.
```